src/calculateRentalCost.js

```diff
@@ -5,6 +5,28 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const daylyRent = 40;
+  let amount = days * daylyRent;
+
+  if (days <= 3) {
+    amount = days * daylyRent;
+
+    // return amount;
+  }
+
+  if (days >= 3 && days < 7) {
+    amount = days * daylyRent - 20;
```

Comment:
Consider using a constant for the discount value to avoid magic numbers. For example, `const MEDIUM_TERM_DISCOUNT = 20;`.